web/client/translations/data.nl-NL.json

```diff
@@ -104,7 +103,7 @@
             "templateFormatInfoAlert2": "Gebruik ${ attribute } om de eigenschappen die je wenst te tonen aan te duiden.",
             "templateFormatInfoAlertExample": "Het id van het object is ${ properties }",
             "templatePreview": "Sjabloon voorbeeld",
-	    "visibilityLimits": {
+	        "visibilityLimits": {
```

Comment:
please do not send changes that are not necessary or errors, check your diffs before.